exercises/stack_02.js

```diff
@@ -0,0 +1,167 @@
+//Stack.push: one
+window.Collections = Object.create(null);
+
+(function(root) {
+    function Stack(arr) {
+        this.arr = arr;
+    };
+    Stack.prototype.push = function(elem, callback) {
+        this.arr.push(elem);
+        callback(this.arr);
+    };
+    // export
+    root.Stack = Stack;
+})(window.Collections);
+
+var stack = new Collections.Stack([1, 2, 3]);
+stack.push(4, console.log);
+
+
+
+//Stack.push: multiple
+window.Collections = Object.create(null);
+
+(function(root) {
+    function Stack(arr) {
+        this.arr = arr;
+    };
+    Stack.prototype.push = function(...callback) {
+        var callback = arguments[arguments.length - 1];
+
+        for (var i = 0; i < arguments.length - 1; i++) {
+            this.arr.push(arguments[i]);
+        }
+        callback(this.arr);
```

Comment:
Don't think callback makes sense here. This method is a little confused. Glad to answer Q's about it. 

If your goal was simply to support the syntax `Stack.push(elem1, elem2, ..., elemN)`, then that actually involves making use of the built-in `arguments` Array-like object, which wasn't covered in my slides, but which you partially used here. In this case, you'd simply leave off the arguments list from the function.